src/software/simulated_tests/validation/terminating_function_validator_test.cpp

```diff
@@ -82,16 +85,17 @@ TEST(TerminatingFunctionValidatorTest,
 {
     ValidationFunction validation_function = [](std::shared_ptr<World> world,
                                                 ValidationCoroutine::push_type& yield) {
-        yield();
+        yield("Test message");
```

Comment:
I think a more descriptive message here be more helpful, applies to all that says "Test message"

Reply:
renamed